### ISSUE TITLE
Spanish vocabulary - how best to internationalise

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#371]
Picking up on conversation from #369 and moving here because it is a distinct issue that needs to be addressed - there are various Spanish language dialects. @arojascastro and @vgayolrs have raised concerns about how best to decide which dialect to use for certain terms ( @mariajoafana ).

If it helps at all, we have the same problem with spelling in the English language team.

In the UK we tend to use 'ise' as in 'internationalise'. Americans use 'ize'. Canadians have their own system. There are other differences as well (mostly spelling, but sometimes idiomatic phrases that sound funny other cultures). 

We decided to go with the dialect of the author. I don't know if that will work in your case. Maybe the dialect of the lead translator? It's up to the three of you.

**Comment by arojascastro** [comment]
Thanks @acrymble for this. I am happy to go with the dialect of the author but  for the sake of the project I would also like to feel free to discuss or question expressions if I am not sure about them.

**Comment by mariajoafana** [comment]
I moved this from the other ticket: 

I think that as long as the translations respect grammar, clarity, and the integrity of the original English lesson we shouldn't worry too much about word choices. If a translator from Spain uses the word "ordenador" as an editor I wouldn't suggest to change it (in Colombia we use "computador") as I believe people both in Latin America and Spain will understand it. @arojascastro I usually use the RAE dictionary which is a good source precisely because It includes words used in both Spain and Latin America.   
There are some stylistic suggestions however that editors make and is up to the translator to follow them or not (the case of ambiente or entorno). As long as they don't compromise the clarity of the text, I think that we can be somehow flexible on this. 
Another issue is the translation of technical terms which sometimes can be challenging. We could start creating a glossary for this so we assure some consistency across lessons in regards to technical words.

**Comment by vgayolrs** [comment]
I I agree with the decision taken in PH about dialectical forms that  @acrymble  mention.
I think we must respect the author's and the translators forms of expression as long they are universally understood.
In the case of "computer", if the author o translator choice "ordenador", "computadora" o "computador", editors and reviewers must respect as, in any way, it's widely understood one or another.
Yesterday @arojascastro  mention two forms for "development enviroment": "ambiente de desarrollo" (as used here: http://programminghistorian.org/es/lecciones/instalacion-mac ) or "entorno de desarrollo". Both are correct and can be used interchangeably, even in the same text.

**Comment by acrymble** [comment]
feel free to discuss in Spanish. I know that must be much easier. Whatever the three of you decide is fine with me. I'm just hoping to show how we can use the tickets to solve discrete tasks and make sure everyone is on the same page.

**Comment by acrymble** [comment]
This seems to be agreed. The author determines the version of Spanish to be used, but the editor/reviewers check to make sure no confusion across cultures is likely.

Has this been incorporated into the editorial guidelines? If so we can close this ticket.

**Comment by vgayolrs** [comment]
No yet incorporated. I will work on this today, but we can close this ticket.

**Comment by acrymble** [comment]
It's a good idea to close tickets only after the action is completed. That way if you get hit by a bus, we know this work is done (and we will be sad - so please don't get hit by a bus).

**Comment by vgayolrs** [comment]
You're right @acrymble .
These are the incorporations:
http://programminghistorian.org/es/guia-para-revisores#traducciones
http://programminghistorian.org/es/guia-editor#nota-sobre-la-revisi%C3%B3n-de-traducciones
I'll have an eye on that bus. But I will wait to close any ticket only when it is complete.
Thanks!
